src/main/java/org/web3j/console/WalletFunder.java

```diff
@@ -36,17 +36,24 @@
     private static final String BASE_URL = "https://rinkeby.faucet.epirus.web3labs.com";
     private static final String USAGE = "fund <destination-address>";
 
-    public static void main(String[] args) {
+    public static void main(IODevice console, String[] args) {
         if (args.length != 1 && args.length != 3) {
             exitError(USAGE);
         }
 
         try {
+            String fund =
+                    console.readLine(
+                            "This command will fund the specified wallet on the Rinkeby testnet. Do you wish to continue? [Y/n]: ");
+            if (!fund.toUpperCase().equals("Y")) {
```

Comment:
I believe traditionally an upper case Y in a Y/n prompt indicates Y being the default option if nothing is entered.

So this should progress if the user types lower or upper case Y, but also progress when nothing is entered (blank).